examples/platform/silabs/SiWx917/SiWx917/wfx_rsi_host.c

```diff
@@ -197,6 +190,25 @@ sl_status_t wfx_connect_to_ap(void)
     return SL_STATUS_OK;
 }
 
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
```

Comment:
@jmartinez-silabs so with --icd flag will enable the `CHIP_CONFIG_ENABLE_ICD_SERVER` right

Reply:
It does, but the define exist in the app dependency, and this is platform code. Rather than creating circular dependency to make to make the define available here, we created a internal silabs platform define which is `SL_ICD_ENABLED`

Reply:
Changed it to SL_ICD_ENABLED